app/models/agents/post_agent.rb

```diff
@@ -85,7 +85,11 @@ def default_options
     end
 
     def working?
-      last_receive_at && last_receive_at > interpolated['expected_receive_period_in_days'].to_i.days.ago && !recent_error_logs?
+      if last_receive_at
+          last_receive_at > interpolated['expected_receive_period_in_days'].to_i.days.ago && !recent_error_logs?
```

Comment:
Overlooked that before, we are using two spaces for indentation, it looks like 4 here.